src/components/ShareView/ShareView.js

```diff
@@ -1,16 +1,49 @@
 import { MDCRipple } from "@material/ripple";
 import shareViewTemplate from "./ShareView.html";
-
-const SONG = {
-  albumName: "Loud",
-  artistName: "by Rihanna",
-  imageSource: "https://image.ibb.co/k4Qc8L/rihanna-loud.jpg"
-};
+import { MusicService } from "../../services/MusicService";
 
 export class ShareViewComponent {
   constructor(mountPoint, props = {}) {
     this.mountPoint = mountPoint;
     this.props = props;
+    this.title = {};
+  }
+
+  fetchAuthors(authorsIds) {
```

Comment:
fetchAuthors, fetchInfoBySong, fetchSongs... these methods are very similar to those in mymusic. Could you please extract them to the MusicService and reuse here and there?

Reply:
Done